Description: disable "groovy patcher"
 The GROOVY-8163 issue was fixed by freeplane upstream by using a library
 "securegroovy" which uses byte-buddy to instrument groovy code.
 GROOVY-8163 is fixed in groovy-2.5.0 and Debian's groovy-2.4.8-2, so this patch
 removes the upstream "securegroovy" invocation.
Author: Felix Natter
Bug: https://issues.apache.org/jira/browse/GROOVY-8163
Last-Update: 2017-09-15
--- a/freeplane_plugin_script/src/main/java/org/freeplane/plugin/script/GroovyScript.java
+++ b/freeplane_plugin_script/src/main/java/org/freeplane/plugin/script/GroovyScript.java
@@ -34,7 +34,7 @@
 import org.freeplane.features.map.NodeModel;
 import org.freeplane.features.mode.Controller;
 import org.freeplane.plugin.script.proxy.ProxyFactory;
-import org.freeplane.securegroovy.GroovyPatcher;
+//import org.freeplane.securegroovy.GroovyPatcher;
 
 import groovy.lang.Binding;
 import groovy.lang.GroovyObject;
@@ -186,7 +186,7 @@
     private static boolean groovyPatched = false; 
     private Script compileAndCache(final ScriptingSecurityManager scriptingSecurityManager) throws Throwable {
     	if(! groovyPatched){
-    		GroovyPatcher.apply(GroovyObject.class);
+	    //GroovyPatcher.apply(GroovyObject.class);
     		groovyPatched = true;
     	}
     	if (compileTimeStrategy.canUseOldCompiledScript()) {
